doc/source/index.md

```diff
@@ -2,200 +2,226 @@
+|:--:|
+|Model parallelism pattern for distributed large model training.|
 
-If you're interested in contributing to Ray, check out our
-[contributing guide for this release](ray-contribute/getting-involved)
-or see the
-[latest version of our contributing guide](https://docs.ray.io/en/latest/ray-contribute/getting-involved.html)
-to read about the contribution process and see what you can work on.
+ See [distributed training examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#distributed-training) for use cases.
 
-## What documentation resource is right for you?
+### Parallel hyperparameter tuning experiments
+Running multiple hyperparameter tuning experiments is a pattern apt for distributed computing because each experiment is independent of one another. Ray Tune handles the hard bit of distributing hyperparameter optimization and makes available key features such as checkpointing the best result, optimizing scheduling, and specifying search patterns.
 
+|<img src="./images/tuning_use_case.png" width="70%" loading="lazy">|
+|:--:|
+|Distributed tuning with distributed training per trial.|
 
-````{panels}
-:container: text-center
-:column: col-lg-6 px-2 py-2
-:card:
+ See [hyperparameter tuning examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#hyperparameter-tuning) for use cases.
 
----
-**Getting Started**
+### Reinforcement learning
+Ray RLlib offers support for production-level, distributed reinforcement learning workloads while maintaining unified and simple APIs for a large variety of industry applications.
 
-<img src="images/getting_started.svg" alt="getting_started" height="40px">
-^^^^^^^^^^^^^^^
+|<img src="./images/rllib_use_case.png" width="70%" loading="lazy">|
+|:--:|
+|Decentralized distributed proximal polixy optimiation (DD-PPO) architecture.|
 
-If you're new to Ray, check out the getting started guide.
-You will learn how to install Ray, how to compute an example with the Ray Core API, and how to use each of Ray's ML libraries.
-You will also understand where to go from there.
+ See [reinforcement learning examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#reinforcement-learning) for use cases.
 
-+++
+### Batch inference on CPUs and GPUs
+Performing inference on incoming batches of data can be parallelized by exporting the architecture and weights of a trained model to the shared object store. Using these model replicas, Ray scales predictions on batches across workers.
 
-{link-badge}`ray-overview/index.html,"Getting Started",cls=badge-light`
+|<img src="./images/batch_inference.png" width="70%" loading="lazy">|
+|:--:|
+|Using Ray AIR's `BatchPredictor` for batch inference.|
 
----
-**User Guides**
+ See [batch inference examples](https://docs.ray.io/en/latest/ray-overview/use-cases.html#batch-inference) for use cases.
 
-<img src="images/user_guide.svg" alt="user_guide" height="40px">
-^^^^^^^^^^^
+### Multi-model composition for model serving
```

Comment:
I think the image for this one is really hard to see. Maybe we can just have parts of it, or mention that you can "click to enlarge" or so.

Reply:
This diagram has a lot going on. A new version is in the works for upcoming serving tutorials, but in the meantime, I can forward along some alternatives.

Reply:
I'll add a note to "Click image to enlarge." in the caption. @emmyscode, I'd love to update it when the new version is available. Thanks!